lessons/2_more_strings_and_functions/lesson.py

```diff
@@ -110,21 +110,39 @@ def my_cool_function(pet_name):
 # A. Use the function named multiply_and_subtract_five. Given that num1 is equal to 7 and num2 is 3,
 #    what do you expect the output of the function to be? Call the function and test it.
 # # ANSWER A GOES HERE
+# # # MH guess: (7*3)-5=16
 # # ANSWER A CODE GOES HERE
+print("the answer to exercise 1A is:",multiply_and_subtract_five(7,3))
 # B. Look at the following function definition  and subsequent code and then 
 #    predict what will be printed by the print function. Uncomment the code
 #    and test your prediction.
-   
-#    def is_double_digits(number):
-#         number = abs(number)
-#         return number >= 10 and number <= 100
+
+def is_double_digits(number):
+    number = abs(number)
+    return number >= 10 and number <= 100
  
-#    print(is_double_digits)
+print(is_double_digits)
 
 # # ANSWER B GOES HERE
+# # MH prediction:
+# # it ensures the number is positive by giving the absolute value of the number,
+# # then you can see if it is between 10 & 100
+# # (where the latter is inclusive and the former is not?)
+# # and you get a yes if it is?
+# # 
+# # except... what is it doing to print the function without giving it any input?
+# # ran it & got:
+# # <function is_double_digits at 0x7fdc4e268820>
```

Comment:
This is correct. So what do you think it means when the function name itself is used without giving it any input (i.e. without calling it)?